src/components/AdvancedSearchBar/AdvancedSearchBar.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <div ref="searchBar" class="relative">
+    <TextInput
+      id="searchBar"
+      v-model="searchTerm"
+      class="min-w-full"
+      :label="t('search.textLabel')"
+      :sr-only-label="true"
+      @focus="visible = true"
+    >
+      <template #iconLeft>
+        <MagnifyingGlass size="l" />
```

Comment:
Same question here to update to the `Icon` component, you will have to add this component to the `Icon` component for that though, let me know if I can help with that